src/Contracts/DocumentsQuery.php

```diff
@@ -84,6 +95,7 @@ public function toArray(): array
             'limit' => $this->limit ?? null,
             'filter' => $this->filter ?? null,
             'fields' => $this->fields(),
-        ], function ($item) { return null != $item || is_numeric($item); });
+            'retrieveVectors' => (null !== $this->retrieveVectors ? (true === $this->retrieveVectors ? 'true' : 'false') : null),
```

Comment:
Hello @norkunas,
For the Get /documents, the parameters are given in the URL, and unfortunately, there is a parsing error if I keep the boolean value because the boolean is transformed into an integer (0 or 1) by default.
```
could not parse `1` as a boolean, expected either `true` or `false`
```

Don't hesitate to suggest any change that could work, I am not a big fan of this.

Reply:
Ah, understood :)